src/main/java/com/parasoft/findings/sonar/TagPrefixMap.java

```diff
@@ -0,0 +1,42 @@
+package com.parasoft.findings.sonar;
+
+import org.sonar.api.rules.CleanCodeAttribute;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TagPrefixMap {
+    public static final Map<String, CleanCodeAttribute> TAG_PREFIX_MAP = new HashMap<>();
+
+    static {
+        TAG_PREFIX_MAP.put("pb", CleanCodeAttribute.LOGICAL);
```

Comment:
@rliu-parasoft  Can you please help to verify the correctness of this map?

Reply:
Will review this part and feedback to Lee